lib/XML/LibXML/Dom.pm

```diff
@@ -0,0 +1,281 @@
+use v6.c;
+
+use XML::LibXML::CStructs :types;
+
+class XML::LibXML::xmlNs is xmlNs {
+
+	method setNext(xmlNs $n) {
```

Comment:
Are there hard tabs or is that just because of the github diff view? Please stick to four space chars either way.